rest/route/github.go

```diff
@@ -667,6 +683,36 @@ func (gh *githubHookApi) requireSigned(ctx context.Context, userRepo data.UserRe
 	return nil
 }
 
+func (gh *githubHookApi) requireApprovals(ctx context.Context, userRepo data.UserRepoInfo, prNum, requiredApprovalCount int) error {
```

Comment:
Can we call this something like `checkPRApprovals` or something?

Reply:
done